capsules/src/apds9960.rs

```diff
@@ -53,7 +53,6 @@ pub static mut BUFFER: [u8; 16] = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
 //                             ^take_meas() callback stored    ^take_meas_int callback stored       ^low thresh           ^high thresh
 
 #[allow(dead_code)]
```

Comment:
what dead code are we allowing here?

Reply:
I just realized this.  It was leftover from some masks I had before but since deleted.  Ill remove it.